test/appium/tests/atomic/chats/test_commands.py

```diff
@@ -47,14 +47,7 @@ def test_network_mismatch_for_send_request_in_1_1_chat(self):
         send_message.next_button.click()
         send_message.sign_transaction()
         self.network_api.wait_for_confirmation_of_transaction(sender['address'], amount, confirmations=15)
-        if chat_1_sender_message.transaction_status.text != 'Confirmed':
-            self.errors.append('Wrong state is shown for outgoing from Ropsten transaction: "Confirmed" is expected, '
-                               'in fact %s ' % chat_1_sender_message.transaction_status.text)
-
-        if chat_2_request_message.transaction_status.text == 'Confirmed':
-            self.errors.append('Sent from Ropsten transaction is shown as confirmed!')
-
-        self.errors.verify_no_errors()
+        [message.transaction_status.wait_for_element_text('Confirmed') for message in (chat_1_sender_message, chat_2_request_message)]
```

Comment:
Please return self.errors.verify_no_errors() here (there are some errors might be appended in the code above)